readme.md (instructions) for db migrations

kodpelle opened the pull request.

kodpelle committed:
readme.md (instructions) for db migrations

WilliamSkogh reviewed (approved):
Riiiktigt fejn läsmig.md

kodpelle committed:
Revise database migration instructions in README

Updated the README to streamline database migration instructions and remove outdated sections.
